.github/workflows/test.yml

```diff
@@ -1,61 +1,66 @@
 name: test
 
 on:
-  push:
-    branches: ["main"]
-  pull_request:
-  merge_group:
+    push:
```

Comment:
If you want to enforce what seems to be non-standard indentation, maybe add a way to enforce this (maybe via `.editorconfig`)? As it is, VS Code will revert this change as soon as it saves this file.